Dockerfile

```diff
@@ -1,37 +1,54 @@
-FROM python:3.6-slim
+FROM python:3.6-slim@sha256:6f39e7dfc5158b351cfea004541fc85898452c2e4cbeb2b36f00c286fc957a88
+
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONPATH=/app/ \
+    PORT=8888
+
+EXPOSE $PORT
 
 # add a non-privileged user for installing and running
 # the application
 RUN groupadd -g 9000 kinto && \
     useradd -M -u 9000 -g 9000 -G kinto -d /app -s /sbin/nologin kinto
 
+# install a few essentials and clean apt caches afterwards
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        git \
+        gcc \
+        libffi-dev \
+        libldap2-dev \
+        libpq-dev \
+        libsasl2-dev \
+        libssl-dev \
+        libxml2-dev \
+        libxslt1-dev
+
+# Clean up apt
+RUN apt-get autoremove -y && \
```

Comment:
The way we did it previously with `RUN buildDeps...` is the right way to keep dev/build requirements out of the Docker.  What you want to do is: 

1. COPY requirements files 
2. RUN pip install ..., uwsgi --build-plugin, apt-get  autoremove ... (all in one command)
3. COPY . /app

That will keep the middle layer, dependency installation much smaller